configs/pipelines/detector_remax_dino_novelty_detection.pipe

```diff
@@ -0,0 +1,139 @@
+
+config _pipeline:_edge
+  :capacity                                    1
+
+config _scheduler
+  :type                                        pythread_per_process
+
+# ==============================================================================
+
+process input
+  :: video_input
+  :video_filename                              input_list.txt
+  :frame_time                                  1
+  :exit_on_invalid                             false
+
+  :video_reader:type                           image_list
+
+  block video_reader:vidl_ffmpeg
+    :time_source                               start_at_0
+  endblock
+
+  block video_reader:image_list
+    :image_reader:type                         vxl
+    :skip_bad_images                           true
+
+    block image_reader:vxl
+      :force_byte                              true
+    endblock
+
+    block image_reader:add_timestamp_from_filename
+      :image_reader:type                       vxl
+
+      block image_reader:vxl
+        :force_byte                            true
+      endblock
+    endblock
+  endblock
+
+process downsampler
+  :: downsample
+  :target_frame_rate                           5
+  :burst_frame_count                           0
+  :burst_frame_break                           0
+  :renumber_frames                             true
+
+connect from input.image
+        to   downsampler.input_1
+connect from input.file_name
+        to   downsampler.input_2
+connect from input.frame_rate
+        to   downsampler.frame_rate
+connect from input.timestamp
+        to   downsampler.timestamp
+
+process detector_input
+  :: image_filter
+  :filter:type                                 vxl_convert_image
+
+  block filter:vxl_convert_image
+    :format                                    byte
+    :force_three_channel                       true
+  endblock
+
+connect from downsampler.output_1
+        to   detector_input.image
+
+# ==============================================================================
+
+process detector1
+  :: image_object_detector
+  :detector:type                               ocv_windowed
+
+  block detector:ocv_windowed
+    :detector:type                             detector_remax_dino
+
+    :mode                                      adaptive
+    :chip_adaptive_thresh                      2000000
+    :scale                                     1.25
+    :chip_width                                1333
+    :chip_height                               800
+    :chip_step_height                          1100
+    :chip_step_width                           600
+    :chip_edge_filter                          -10
+    :batch_size                                1
+
+    block detector:detector_remax_dino
+      # Network config, weights, and names
+      relativepath net_config  =               models/DINO_4scale_swin.py
+      relativepath weight_file =               [-MODEL-WEIGHTS-FILE] # ../../../../../pretrained_models/dino/pytorch_dino.pth
```

Comment:
Fix paths

Reply:
What do you mean here? When we added the weights file parameter to the convnext model we used the [-MODEL-WEIGHTS-FILE] placeholder. What should go here instead?

Reply:
Pipeline files in the 'configs/pipelines' folder should be default pipelines which work without modification, if this is a template file with values that get filled in by a training routine, it should be in the 'templates' folder not here. Also the comment should be removed #../.../../..   as that has no relevance outside of 1 specific system, and I doubt the find replace string is something like [-MODEL-WEIGHTS-FILE] instead of [-MODEL-WEIGHTS-FILE-] or something symmetrical

Reply:
I added the direct filepaths, I've been using the config/models/ folder to hold the config and checkpoint files. Is that the best place? It seems to be in the git ignore, so I assume I'll have to add the models somewhere else that will get pulled from.